shared/css/gdpr-eu-dialog.scss

```diff
@@ -0,0 +1,106 @@
+@import 'color';
+@import 'font';
+
+#gdpr-eu-dialog {
+  position: fixed;
+  top: 0px;
```

Comment:
I'm confused by this - in the screenshots you've got a dialog in the middle of the page, but it seems like it'd sit at the very top of the viewport here?  What rule am I missing that positions the dialog?

Reply:
`margin: 5%;` maybe? It starts 5% of the way down the page and everything scrolls behind it.